packages/RichTextEditing-Core.package/RichTextStructureTemplate.class/class/textStructureByIndentifier..st

```diff
@@ -0,0 +1,5 @@
+initial structures
```

Comment:
Categorization may be improved with a text-structure category

Reply:
Don't this category already more or less only contain text structure definitions?